lib/importproject.cpp

```diff
@@ -327,13 +310,14 @@ void ImportProject::FileSettings::parseCommandInclude(const std::string& command
 
 void ImportProject::FileSettings::parseCommandSystemInclude(const std::string& command,  std::string::size_type *pos)
 {
-  const bool skipBackSlash = false;
-  const std::string isystem = readUntil(command, pos, " ", skipBackSlash);
-  systemIncludePaths.push_back(isystem);
+    const bool skipBackSlash = false;
```

Comment:
Where does this reformatting come from?

Reply:
@amai2012 I used ` astyle` and  the `runastyle` script